src/browser/settings.jsx

```diff
@@ -56,6 +71,14 @@ var SettingsPage = React.createClass({
       var currentWindow = remote.getCurrentWindow();
       currentWindow.setAutoHideMenuBar(config.hideMenuBar);
       currentWindow.setMenuBarVisibility(!config.hideMenuBar);
+
+      var autostart = this.state.autostart;
+      appLauncher.isEnabled().then(function(enabled) {
```

Comment:
Sorry for my poor English, I couldn't understand.
Point of my question was "Is there any problem for executing enable/disable every time when clicking save button?" `if` condition looks a little complex because of `isEnabled()`